modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/congrats/index.html

```diff
@@ -0,0 +1,18 @@
+<div class="container congrats">
```

Comment:
If this class is not used, please remove it.
```suggestion
<div class="congrats">
```